code/modules/mob/mob_helpers.dm

```diff
@@ -331,19 +331,86 @@
+
+	if(prob(50)) //stuff we only want replaced this way SOME of the time, the alternative goes under ELSE
+		message = replacetext(message, "eau", "eu") //beutiful
+		message = replacetext(message, "ie", "ei")
+		message = replacetext(message, "tab", "tib") //tible, comfortible
+		message = replacetext(message, "'re", "re")
+		message = replacetext(message, "help", "halp")
+		message = replacetext(message, "wood", "woody")
+		message = replacetext(message, "ain", "ian")
+		if(prob(25)) //13%
+			message = replacetext(message, " am ", " ")
+			message = replacetext(message, " is ", " ")
+			message = replacetext(message, " are ", " ")
+	else //these prob checks are per sentence, so saying friend 100 times in one sentence will have all 100 look the same instead of mix and match
+		message = replacetext(message, "eau", "u")
+		message = replacetext(message, "friend", "fren")
+		message = replacetext(message, "tab", "teib") //teible, comforteible
+		message = replacetext(message, "you're", "your")
+		message = replacetext(message, "help", "helb")
+		message = replacetext(message, "y ", "i ")
+		message = replacetext(message, "y,", "i")
+		message = replacetext(message, "captain", "COMDOM")
+	if(prob(70))
+		var/regex/duplicate = new("(\\w)(?=\\1)", "g")
+		message = replacetext(message, duplicate, "") //duplicate letters into one letter, for all words
+	if(prob(40)) //occasionally replaces one word with umm.
+		var/list/words = splittext(message, " ")
+		words[rand(0, words.len)] = pick("um,", "umm,", "uh,", "uhh,")
+		message = jointext(words, " ")
+			
+	//specific words we want replaced every time, mainly references to the actual brain damage lines, so they stay consistent with brain damage regular speech
+	message = replacetext(message, "stop", "sotp")
+	message = replacetext(message, "retarded", "rarted")
+	message = replacetext(message, "george", "[pick("joerge", "george", "gorge", "gdoruge")]")
+	message = replacetext(message, "melon", "[pick("mellens", "melons", "mwrlins")]")
+	message = replacetext(message, "oxygen", "oxigen")
+	message = replacetext(message, "evolve", "evoluate")
+	message = replacetext(message, "unrealistic", "unralistic")
+	message = replacetext(message, "round", "rouns")
+	message = replacetext(message, "roles", "rolls")
+	message = replacetext(message, "worst", "wurst")
+	message = replacetext(message, "worse", "wurse")
+	message = replacetext(message, "vegan", "vegana")
+	message = replacetext(message, "boob", "bobs")
 	message = replacetext(message, "grief", "griff")
 	message = replacetext(message, "space", "spess")
 	message = replacetext(message, "carp", "crap")
 	message = replacetext(message, "reason", "raisin")
-	if(prob(50))
+	message = replacetext(message, "engineer", "engiener")
+	message = replacetext(message, "pinpointer", "pin pointner")
+	message = replacetext(message, "singularity", "scrungulartiy")
+	message = replacetext(message, "based", "BASTE")
+	message = replacetext(message, "hos", "HITLER")
```

Comment:
perhaps use `" hos ", " HITLER "` so you don't turn "those" into "tHITLERe"
though a borer hitlert sounds pretty funny